spec/migrations/data/20190617191109_move_awx_credentials_to_authentications_spec/awesome_stubs.yaml

```diff
@@ -0,0 +1,237 @@
+    SECRET_KEY: ecad5764714a254a619d74ccc1c4387b
+    PRIMARY_KEY: '3'
+  :output: |
+    -----BEGIN RSA PRIVATE KEY-----
```

Comment:
Incase anyone is curious, I provided this data, and just used a SSH key that I generated for a `vagrant` environment I was using for `EmbeddedAnsible` testing.  Generated at the top of my vagrant file by doing the following:

```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :

# Setup keys for inter-VM communitcation to the :share VM
_, priv_key, pub_key = [nil, nil, nil]
unless File.exist? "share.id_rsa" and File.exist? "share.id_rsa.pub"
  require "vagrant/util/keypair"

  _, priv_key, pub_key = Vagrant::Util::Keypair.create

  File.write "share.id_rsa",     priv_key
  File.write "share.id_rsa.pub", pub_key
end
```

Private keys used here are from the `shared.id_rsa` file, which I will be deleting shortly.

... also, this message will self destruct in 5 seconds. :boom: